tools/partitions/large_spiffs_16MB.csv

```diff
@@ -3,4 +3,4 @@ nvs,      data, nvs,     0x9000,  0x5000,
 otadata,  data, ota,     0xe000,  0x2000,
 app0,     app,  ota_0,   0x10000, 0x480000,
 app1,     app,  ota_1,   0x490000,0x480000,
-spiffs,   data, spiffs,  0x910000,0x6F0000,
+spiffs,   data, spiffs,  0x910000,0x6EFC00,
```

Comment:
@CoryCharlton - please test it with:

spiffs, data, spiffs, 0x910000,0x6EFFFC,

Reply:
Confirm they work and committed the updated files.